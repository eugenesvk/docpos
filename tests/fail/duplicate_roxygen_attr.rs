use roxygen::*;

#[roxygen]
/// here are some comments
#[roxygen]
/// and some more
#[arguments_section]
/// and some more
/// but this next argument section should not be here
#[arguments_section]
pub fn add(
    /// some comments
    first: i32,
    second: i32,
) -> i32 {
    first + second
}

pub fn main() {}
