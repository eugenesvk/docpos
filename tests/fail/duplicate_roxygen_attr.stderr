error: Duplicate attribute. This attribute must only appear once.
 --> tests/fail/duplicate_roxygen_attr.rs:5:1
  |
5 | #[roxygen]
  | ^^^^^^^^^^
